.github/workflows/publish.yml

```diff
@@ -25,6 +28,19 @@ jobs:
       id-token: 'write'
 
     steps:
+      - name: Ensure that a Jira ID is present in the commit message
```

Comment:
TOL: I wonder whether we can have a check on PRs to make sure their titles are well formed, in addition to this check which happens after the merge?

Reply:
Edit: never mind, copilot did it in one tab-complete: https://github.com/DataBiosphere/cbas/pull/155